### PR TITLE
Rebuild base image: k8s job -> Github Action

### PR DESCRIPTION
**Pull request by jpopelka**
So far we've been using a cron job in Openshift to trigger a build in Docker Hub.
With this, we use a Github Action to build in Github and push to Quay.io.

### CODE REVIEW

**Review by TomasTomecek — APPROVED** [review]
very nice!!!

File: .github/workflows/base-image-rebuild.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: ${{ steps.build-image.outputs.image }}
+          tags: ${{ steps.build-image.outputs.tags }}
+          registry: packit
+          username: packit+base_image_builder
+          password: ${{ secrets.QUAY_BASE_IMAGE_BUILDER_TOKEN }}
+
+      - name: Use the image
+        run: echo "New image has been pushed to ${{ steps.push-to-quay.outputs.registry-paths }}"
```
Comment (csomh) [inline .github/workflows/base-image-rebuild.yml:37]: Is this needed? I would expect that this information is available in the `push-to-registry` action logs.

File: cron-jobs/README.md [hunk 2]
```diff
@@ -1,13 +1,3 @@
...
-
-## rebuild-base-image
-
-The `rebuild-base-image` CronJob is rebuilding the base image using a DockerHub webhook. The webhook is stored in the secrets repo.
-
-Rebuild-base-image is deployed using root Makefile:
-
-```shell script
-make deploy-rebuild-base-image
-```
```
Comment (csomh) [inline cron-jobs/README.md:13]: Maybe mention that rebuilding is done as part of a scheduled GitHub workflow.

**Review by csomh — COMMENTED** [review]
You don't know how happy I am that this is happening :rocket: :partying_face: :champagne:

File: .github/workflows/base-image-rebuild.yml (hunk 1 above)
Comment (jpopelka) [reply .github/workflows/base-image-rebuild.yml]: no, it's not, I just copied it from [example](https://github.com/marketplace/actions/push-to-registry#examples)

File: cron-jobs/README.md (hunk 2 above)
Comment (jpopelka) [reply cron-jobs/README.md]: added

**Comment by jpopelka** [comment]
I finally got it working in my fork, please re-review

**Review by csomh — APPROVED** [review]

**Pull request merged**